src/ConductorSharp.Toolkit/Service/ScaffoldingService.cs

```diff
@@ -0,0 +1,286 @@
+﻿using ConductorSharp.Client.Model.Common;
+using ConductorSharp.Client.Service;
+using ConductorSharp.Engine.Util;
+using ConductorSharp.Toolkit.Util;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace ConductorSharp.Toolkit.Service
+{
+    public class ScaffoldingService : IScaffoldingService
+    {
+        private readonly IMetadataService metadataService;
+        private readonly ILogger<ScaffoldingService> logger;
+        private readonly ScaffoldingConfig config;
+
+        public ScaffoldingService(IMetadataService metadataService, IOptions<ScaffoldingConfig> options, ILogger<ScaffoldingService> logger)
+        {
+            this.metadataService = metadataService;
+            this.logger = logger;
+            config = options.Value;
+        }
+
+        public async Task Scaffold()
+        {
+            var workflowDefinitions = await metadataService.GetAllWorkflowDefinitions();
+            var workflowCollectionBuilder = new StringBuilder();
+            foreach (var taskDefinition in workflowDefinitions)
+            {
+                var workflowClass = await CreateWorkflowClass(taskDefinition);
+
+                if (workflowClass != null)
+                    workflowCollectionBuilder.Append(workflowClass);
+            }

```

Comment:
This is probably a typo for workflowDefinition
```suggestion
            foreach (var workflowDefinition in workflowDefinitions)
            {
                var workflowClass = await CreateWorkflowClass(workflowDefinition);

                if (workflowClass != null)
                    workflowCollectionBuilder.Append(workflowClass);
            }

```

Reply:
Done